### PR TITLE
chore(trading): remove quadratic slippage factor

### LINKED ISSUE #5551: Update market info to support removal of quadratic slippage factor
## The Chore

The "Quadratic Slippage Factor" in the "Margin scaling factors" section of the market info will be removed.  

## Tasks

- [ ] Check usage of these values and make sure they aren't used anywhere else
- [ ] Update the market info component to handle it
- [ ] Check Console is happy 
- [ ] Check Explorer is happy

### PR DESCRIPTION
**Pull request by MadalinaRaicu**
# Related issues 🔗

Closes #5551

# Description ℹ️

Remove quadratic slippage factor from market info.

### CODE REVIEW

**Review by mattrussell36 — APPROVED** [review]

**Review by daro-maj — CHANGES_REQUESTED** [review]
Please revert quadratic slippage factor in explorer and governance tests, currently they are failing and this is probably related to https://github.com/vegaprotocol/vega/issues/10343

File: apps/governance-e2e/src/fixtures/proposals/new-market.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "decimalPlaces": "5",
     "positionDecimalPlaces": "5",
     "linearSlippageFactor": "0.001",
-    "quadraticSlippageFactor": "0",
+    "quadraticSlippageFactor": "0.0",
```
Comment (MadalinaRaicu) [inline apps/governance-e2e/src/fixtures/proposals/new-market.json:6]:
```suggestion
    "quadraticSlippageFactor": "0",
```

File: apps/governance-e2e/src/fixtures/proposals/successor-market.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
     "decimalPlaces": "5",
     "positionDecimalPlaces": "5",
     "linearSlippageFactor": "0.001",
-    "quadraticSlippageFactor": "0",
+    "quadraticSlippageFactor": "0.0",
```
Comment (MadalinaRaicu) [inline apps/governance-e2e/src/fixtures/proposals/successor-market.json:6]:
```suggestion
    "quadraticSlippageFactor": "0",
```

File: apps/governance-e2e/src/fixtures/proposals/update-market.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "linearSlippageFactor": "0.001",
-  "quadraticSlippageFactor": "0",
+  "quadraticSlippageFactor": "0.0",
```
Comment (MadalinaRaicu) [inline apps/governance-e2e/src/fixtures/proposals/update-market.json:3]:
```suggestion
  "quadraticSlippageFactor": "0",
```

**Review by daro-maj — APPROVED** [review]

**Review by jeremyletang — APPROVED** [review]

**Pull request merged**